python/mxnet/module/base_module.py

```diff
@@ -740,14 +760,19 @@ def install_monitor(self, mon):
     ################################################################################
     # Computations
     ################################################################################
-    def prepare(self, data_batch):
+    def prepare(self, data_batch, row_id_generator=None):
```

Comment:
generator has specific meaning in python (i.e. one-time iterable). maybe use `_fn` suffix to note this is a callback function?